03_letters/jecofang/letter_frequency_test.go

```diff
@@ -0,0 +1,74 @@
+package main
+
+import (
+	"bytes"
+	"strconv"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+)
+
+func TestMainOutput(t *testing.T) {
+	// Given
+	r := require.New(t)
+	var buf bytes.Buffer
+	out = &buf
+
+	// When
+	main()
+
+	// Then
+	expected := strconv.Quote(`a:2
+b:1
```

Comment:
We debated this one back and forth. It's a line call, but on balance, it's probably better to write `"a:2\nb:1"`.